lldb/include/lldb/lldb-defines.h

```diff
@@ -96,6 +96,7 @@
 #define LLDB_INVALID_QUEUE_ID 0
 #define LLDB_INVALID_CPU_ID UINT32_MAX
 #define LLDB_INVALID_WATCHPOINT_RESOURCE_ID UINT32_MAX
+#define LLDB_INVALID_PORT_NUMBER 0
```

Comment:
Port numbers aren't an lldb invention, so I don't think we should be putting this into the lldb's public (and forever immutable) API.